hack/runtime-migrator/README.md

```diff
@@ -0,0 +1,65 @@
+  -output-path=/tmp/ \
+  -dry-run=true \
+  -input-type=json 1> /tmp/stdout.txt 2> /tmp/stderr.txt
+```
+
+The above **execution example** will: 
+1. take the stdin input (json with runtimeIds array)
+1. proceed only with Runtime Custom Resources creation for clustest listed in the input 
+1. save generated Runtime Custom Resources yamls in `/tmp/` directory. They will not be applied on the KCP cluster (`dry-run` mode)
+1. send logs and errors to `/tmp/stderr.txt`
+1. send json output to `/tmp/stdout.txt`
+
+### Json output example
+
+```json
+[
+    {
+        "runtimeId": "aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa",
+        "shootName": "shoot-name1",
+        "status": "Success",
+        "pathToCRYaml": "/tmp/shoot-shoot-name1.yaml"
+    },
+    {
+        "runtimeId": "",
+        "shootName": "shoot-name2",
+        "status": "Error",
+        "errorMessage": "Shoot networking pods is nil"
+    }
+]
+```
+
+## Configurable Parameters
+
+This table lists the configurable parameters, their descriptions, and default values:
+
+| Parameter | Description                                                                                                                                                                                                                | Default value                  |
+|-----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------|
+| **kcp-kubeconfig-path** | Path to the Kubeconfig file of KCP cluster.                                                                                                                                                                                | `/path/to/kcp/kubeconfig`      |
+| **gardener-kubeconfig-path** | Path to the Kubeconfig file of Gardener cluster.                                                                                                                                                                           | `/path/to/gardener/kubeconfig` |
+| **gardener-project-name** | Name of the Gardener project.                                                                                                                                                                                              | `gardener-project-name`        |
+| **output-path** | Path where generated yamls will be saved. Directory has to exist.                                                                                                                                                          | `/tmp/`                        |
+| **dry-run** | Dry-run flag. Has to be set to **false**, otherwise migrator will not apply the Custom Resources on the KCP cluster.                                                                                                       | `true`                         |
```

Comment:
```suggestion
| **dry-run** | Dry-run flag. Has to be set to **false**, otherwise migrator will not apply the CRs on the KCP cluster.                                                                                                       | `true`                         |
```